rpm/template.spec.em

```diff
@@ -31,6 +36,9 @@ Source0:        %{name}-%{version}.tar.gz
 %autosetup
 
 %build
+# Needed to bootstrap since the ros_workspace package does not yet exist.
+export PYTHONPATH=@(InstallationPrefix)/lib/python%{python3_version}/site-packages
```

Comment:
out of curiosity, this template will end up with a bash script? If that is the case, why we need to export multiple times the PYTHONPATH?

Reply:
Each section (build/install/check) will be a separate script launched by a separate shell. Only RPM macros carry between sections, as they are processed at the document (spec file) level.